LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2014 Fusonic GmbH (http://www.fusonic.net)
```

Comment:
Copyright (c) 2018 Fusonic GmbH (https://www.fusonic.net) -> would be much better